lib/assessment.js

```diff
@@ -40,6 +89,20 @@ var newArray2 = function newArray2(length, defaultsFunction) {
 // the object passed in as an argument or the object created
 var addMethod = function addMethod(propertyName, method, obj) {
 
+  //need case where obj is not defined yet
+  if (obj === undefined) {
+
+    //creates obj incase one passed through doesn't exist yet
+    obj = {};
+  }
```

Comment:
Very good! Here's another way to do it:
`obj = obj || {};`

Can you see why that works?